### ISSUE TITLE
VTF import in linux imports blue

### ISSUE THREAD

**Issue by Urammar** [lasa01/Plumber#193]
**Describe the bug**
Importing VTF files saturates them blue for some reason?

Linux on latest blender

**Comment by Red-Teapot** [comment]
Yeah, I'm getting the same issue. Instead of `#d16a87` (checked via VTFEdit running under Wine) I'm getting `#d16ad7` just as an example

**Comment by lasa01** [comment]
That's weird. What Linux distro is this and what files are you trying to
import from what game? Or can you upload an example file here with the
issue? I haven't tested this on Linux for a while but it has worked
correctly before.

**Comment by Red-Teapot** [comment]
I'm running NixOS unstable, Blender 4.5.3, trying to import a Garry's Mod map (this one: https://steamcommunity.com/sharedfiles/filedetails/?id=3138642703; as a note, the actual assets are stored in the gm_voidplaces map which is required to run vp_voidmall)

I'm noticing three kinds of behavior, depending on the texture:
- Blue channel gets oversaturated
- Some pixels become light blue-ish color
- The texture is imported correctly

Otherwise, the addon works fine, I think.

I've created a ZIP with three VTFs, each behaving differently: [VTFs.zip](https://github.com/user-attachments/files/23423392/VTFs.zip)
- `pb_dc_mfull_concretefloor_f2` gets weird blue pixels
- `pb_decal_wcdiscovery` imports just fine
- `pb_vmall_innophoto_idle` gets the oversaturated blue channel

**Comment by lasa01** [comment]
There was a bug with integer data type definitions on Linux and MacOS. This build should have this fixed. Can you test if this build works well? Remember to restart Blender after updating.

[plumber-1.2.0-linux_x64.zip](https://github.com/user-attachments/files/23506113/plumber-1.2.0-linux_x64.zip)

**Comment by Red-Teapot** [comment]
Yeah, the new build looks fine, thank you!

**Comment by lasa01** [comment]
Should be fixed in the latest release [here](https://github.com/lasa01/Plumber/releases/tag/v1.2.1)
